resource-managers/kubernetes/architecture-docs/submission-client.md

```diff
@@ -0,0 +1,220 @@
+---
+layout: global
+title: Implementation of Submitting Applications to Kubernetes
+---
+
+
+Similarly to YARN and Standalone mode, it is common for Spark applications to be deployed on Kubernetes through the
+`spark-submit` process. Applications are deployed on Kubernetes via sending YML files to the Kubernetes API server.
```

Comment:
`s/YML/YAML`. Also this is probably not accurate. Applications are deployed on Kubernetes by creating Kubernetes API objects via the API server. Such Kubernetes API objects are typically declared in YAML files.